crates/holochain_zome_types/src/dna_compat.rs

```diff
@@ -0,0 +1,44 @@
+//! DNA compatibility parameters
+
+use holo_hash::DnaHashB64;
+use holochain_serialized_bytes::prelude::*;
+
+/// Extra parameters that contribute to determining the DNA hash.
+/// They are set by the conductor at install time and cannot be specified
+/// by the DNA developer.
+/// They represent different aspects of networking compability.
```

Comment:
```suggestion
/// They represent different aspects of networking compatibility.
```

Reply:
In light of the office hours, I'm probably going to scrap this PR altogether!

Reply:
I was gonna say that at first actually.